site/docs/pages/wallet/wallet.mdx

```diff
@@ -476,6 +476,37 @@ export default OnchainProviders;
 
 ::::
 
+## Examples
+
+### With Sign In With Ethereum (SIWE)
+
+To use SIWE with OnchainKit, you can use the `onConnect` prop in the `<ConnectWallet />` component. This will trigger the SIWE prompt when the user connects their wallet.
```

Comment:
nit, can we spell out `Sign In With Ethereum` on the first use of the acronym and have it link out to SIWE docs?

Reply:
Good point